app/forms.py

```diff
@@ -67,3 +67,18 @@ class GenerateReportForm(Form):
                       description='yyyy-mm-dd', validators=[Optional()])
     end = DateField('End Date:', format="%Y-%m-%d",
                     description='yyyy-mm-dd', validators=[Optional()])
+
+
+class CreateProjectForm(Form):
+    uri = StringField('URI:')
```

Comment:
I'm not sure where it would go but should there be some validation that this is a uri and not just a string?